net/art/stride_table.go

```diff
@@ -69,61 +69,50 @@ const (
 	lastHostIndex = 0b1_1111_1111
 )
 
-// getChild returns the child strideTable pointer for addr (if any), and an
-// internal array index that can be used with deleteChild.
-func (t *strideTable[T]) getChild(addr uint8) (child *strideTable[T], idx int) {
-	idx = hostIndex(addr)
-	return t.entries[idx].child, idx
+// getChild returns the child strideTable pointer for addr (if any).
```

Comment:
(or nil if none)

?

Reply:
Replaced the various (if any)s with better words.